cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -81,16 +82,13 @@ <h2 id="interested">Interested in Applying?</h2>
                          Social Media icons are slightly high
                          relative to Interested in Applying heading. #}
                 {{ social_media.render( {
-                    'title':            page.job_title,
-                    'twitter_hashtags': page.job_twitter_hash_tag or 'usajobs'
+                    'title':            page.title,
+                    'twitter_hashtags': page.twitter_hash_tag or 'usajobs'
```

Comment:
@schaferjh we didn't discuss this as part of the data model. Is per-job customization still needed here?